app/services/slug_setup.rb

```diff
@@ -0,0 +1,58 @@
+class SlugSetup
+    
+    attr_reader :name 
+
+    def self.setup(name)
```

Comment:
you might want to read this as regards naming the service and its main method:

http://blog.silvabox.com/musings-on-service-objects/

also see the discussion in https://github.com/AgileVentures/LocalSupport/pull/306 and look at how I designed the CreateProposedOrganisationEdit service